app/models/user.rb

```diff
@@ -1,8 +1,17 @@
 class User < ApplicationRecord
   has_secure_password
+
   has_many :reservations, dependent: :destroy
+  has_many :equipments
+
+  has_one_attached :image
 
-  validates :email, presence: true, uniqueness: true
+  validates :email, uniqueness: true
   validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
-  validates :username, presence: true, uniqueness: true
+  validates :username, uniqueness: true
```

Comment:
Hey mate. I understand you deleted the ```presence: true``` because the null is handled by the database. However, this allows users to enter a blank username which is not suitable either.

I'm also thinking we should add the presence validation for the name property as well, what do you think?